src/app/home/components/PromotionModal/PromotionModal.tsx

```
@@ -0,0 +1,124 @@
+                  onClick={() => setCurrentImageIndex(currentImageIndex - 1)}
+                >
+                  <Icon icon="ChevronSquare" rotate={-90} cursor="pointer" color={'#ffffff1a'} />
+                </button>
+              )}
+
+              {currentImageIndex < images.length - 1 && (
+                <button
+                  className={`${navigationButton} ${navigationRightButton}`}
+                  onClick={() => setCurrentImageIndex(currentImageIndex + 1)}
+                >
+                  <Icon icon="ChevronSquare" rotate={90} cursor="pointer" color={'#ffffff1a'} />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)